src/node/routes/index.ts

```diff
@@ -55,6 +55,9 @@ export const register = async (
       })
     })
   })
+  server.on("close", () => {
+    heart.dispose()
+  })
```

Comment:
Is this related to this PR? I thought I saw something about the heart beat error - guessing this helps with that?

Reply:
See the commit this was introduced in. Always review by commit otherwise you'll miss the context behind changes. This change is from `heart.ts: Fix leak when server closes`.

Reply:
These three lines fixed the leak preventing the tests from returning by disabling the heartbeat timer.